capstone-app/src/app/modules/maps/map/map.component.spec.ts

```diff
@@ -27,29 +25,102 @@ describe('MapComponent', () => {
     expect(fixture.componentInstance).toBeTruthy();
```

Comment:
Optional: These kinds of tests are just placeholders when you don't have anything else running, but they're not really necessary. You can remove this test now if you want.